### DISCUSSION TITLE
Moving .kryptor files to .config/kryptor

### DISCUSSION THREAD

**Discussion by ariek48** [samuel-lucas6/Kryptor#58] [Ideas]
The .kryptor folder which contains the keys resides in the ~/ folder . This at least in MacOs. I havent tried it yet in Linux.
I know i could specify the location of the keys when encrypting and decrypting. 

But can it just follow the XDG Base Directory Specification . It makes backing up much easier and its less cluttered in the home folder

Environment variables
$XDG_CONFIG_HOME defines the base directory relative to which user-specific configuration files should be stored. If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used.

**Comment by samuel-lucas6** [comment]
Fair suggestion. It's there on Linux too. That was chosen because it's consistent and easily accessible across platforms, unlike other `Environment.SpecialFolder` paths. It also matches [Minisign](https://jedisct1.github.io/minisign/).

`Environment.SpecialFolder.ApplicationData` does what you're asking but would use `%APPDATA%` on Windows, which I suppose isn't a big deal. However, this would be annoying to change because it would require checking two locations and copying the old directory, so I'm inclined to leave it as it is. On my machine, dotnet, nuget, Rider, and VirtualBox all use the home directory, so this can't be unheard of.
